packages/core/config/babel-plugin-redwood-cell.js

```diff
@@ -0,0 +1,70 @@
+module.exports = function ({ types: t }) {
+  const names = [
```

Comment:
```suggestion
  // These are the expected named exports from a Cell file.
  const EXPORTS_FROM_CELL = [
```

Reply:
Maybe `CELL_EXPORTS` is a better name 🤷‍♂️

Reply:
I like `EXPORTS_FROM_CELL` better!